packages/amplify-nodejs-function-runtime-provider/src/utils/invoke.ts

```diff
@@ -6,12 +6,19 @@ import path from 'path';
 export function invoke(options: InvokeOptions): Promise<any> {
   return new Promise((resolve, reject) => {
     try {
+      let data: string = '';
       const lambdaFn = fork(path.join(__dirname, 'execute.js'), [], {
```

Comment:
This might be a good time to finally migrate this function to use [execa](https://www.npmjs.com/package/execa) instead of child_process. It should make all of this cleaner and natively supports promises